<launch>

    <!-- list of arguments -->
    <arg name="skill_name"  default="pick" />
    <arg name="model_type"  default="visuomotor_bc" />
    <arg name="train_type"  default="reg" />

    <!-- HAL SKLIL -->
    <node name="hal_skills" pkg="stretch_learning" type="hal_skills.py" output="screen"
        args ="--skill_name $(arg skill_name) --model_type $(arg model_type) --train_type $(arg train_type)">
    </node>

</launch>
